src/Interceptor.ts

```diff
@@ -1,11 +1,14 @@
-import { Debugger, debug } from 'debug'
+import type { Debugger } from 'debug'
+import pkg from 'debug'
```

Comment:
Perhaps let's name it something else than a generic `pkg`? In fact, maybe we should depict the reason for this kind of import in the name itself:

```js
import debugCjsImport from 'debug'

const debug = debugCjsImport
```

What do you think about this?

Reply:
Sure - Makes sense.